lib/app/sequence_base.rb

```diff
@@ -583,7 +588,8 @@ def test_resources(resource_type)
             validate_resource(resource_type, resource, p)
           else
             warn { assert false, 'No profiles found for this Resource' }
-            resource.validate
+            issues = Inferno::RESOURCE_VALIDATOR.validate(resource, versioned_resource_class)
+            issues[:fatals].concat issues[:errors]
```

Comment:
This logic seems like it's duplicated a lot. Would it be feasible to add a method like `validation_errors` that does the validation and returns the fatal/error issues already concatenated?

Reply:
Yeah, I could look into doing that. Would cut down on repetitive code.

Reply:
Update: after talking with Reece, I decided to condense the number of returned messages from the validators down to just `errors` and `warnings`. In the HL7 Validator, both `fatals` and `errors` are treated as `errors`, and `warnings` and `information` are treated as `warnings`. This more closely matches how we treat the messages in Inferno currently, while leaving open the ability to have more granular error reporting in the future.